code/compute_fc/compute_fc.py

```diff
@@ -0,0 +1,883 @@
+""" Python script to denoise and aggregate timeseries before computing functional
```

Comment:
```suggestion
""" Python script to denoise and aggregate timeseries and, using the latter, compute functional
```
To me, it was confusing because I understood that you did not compute FC in this script